src/components/ProjectCard/ProjectCardSidePanel.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import { StatusLabel } from 'hds-react/components/StatusLabel';
+import { IconSaveDiskette } from 'hds-react/icons';
+import { SideNavigation } from '../shared';
+import { useAppSelector } from '@/hooks/common';
+import { RootState } from '@/store';
+import { useTranslation } from 'react-i18next';
+
+const ProjectCardSidePanel: FC = () => {
+  const updated = useAppSelector((state: RootState) => state.projectCard.updated);
+  const { t } = useTranslation();
+  const navItems = [
+    {
+      route: '#basics',
+      label: t('nav.basics'),
+    },
+    { route: '#schedule', label: t('nav.schedule') },
+  ];
+
+  return (
+    <div className="side-panel-container">
+      <div className="nav-container">
+        <SideNavigation navItems={navItems} />
+      </div>
+
+      {updated && (
+        <div className="label-container">
+          <div className="side-nav">
+            <StatusLabel
+              type="success"
+              iconLeft={<IconSaveDiskette />}
+            >{`Tallennettu: ${updated}`}</StatusLabel>
```

Comment:
Should we use here translation instead of direct text "Tallennettu"?

Reply:
Fixed